### DISCUSSION TITLE
Making cursor flashing when editing values/text?

### DISCUSSION THREAD

**Discussion by MiqViq** [Spirik/GEM#90] [General]
Does GEM have an option to set the alternating "flashing" effect on cursor?
This would be nice when text (or number values) are under editing. 
Cursor type could be selectable between  block/underscore.
And flashing on/off with adjustable flashing rate.

If this is not yet possible then this will be a request for new enhancement, I guess...

**Comment by Spirik** [comment]
Hi, @MiqViq!

Flashing version of cursor is not currently supported, and, to be completely honest, I am not sure if it is really worth it: implementing it may add additional performance overhead because of additional timers and need to constantly redraw portion of the screen. (And overall size of the library is currently of a big concern - so I am trying to further avoid excessive feature bloat).

Besides, I believe, that current static cursor should give enough feedback to the user regarding "editability" of a currently selected item (in edit mode interactive elements are drawn with inverted colors, be it single digit/character, or option select item) - adding flashes to it won't improve communication with user much.

That being said, I am open to experimenting with this idea as a side feature, that is possible to implement on per project basis by overriding some of the core drawing routines of GEM in user's own sketch (in [Advanced Mode](https://github.com/Spirik/GEM?tab=readme-ov-file#advanced-mode)).

**Reply by MiqViq** [reply]
@Spirik, thanks for a quick response!

If I really need that functionality I will try it with Advanced Mode.
